src/OpenRiaServices.Client.HttpDomainClient/Framework/BinaryHttpDomainClientFactory.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+
+namespace OpenRiaServices.Client.HttpDomainClient
+{
+    public class BinaryHttpDomainClientFactory
+        : DomainClientFactory
+    {
+        private Func<HttpClient> httpClientFactory;
+
+        public BinaryHttpDomainClientFactory()
+            : this(new HttpClientHandler()
```

Comment:
i am considering to remove this since not all platforms (blazor) support the cookiecontainer and it might be better if the end user configures it to match the  applications requirement instead of getting runtime exceptions